exporter/debugexporter/factory.go

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package debugexporter // import "go.opentelemetry.io/collector/exporter/debugexporter"
+
+import (
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/loggingexporter"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr                   = "debug"
+	defaultSamplingInitial    = 2
+	defaultSamplingThereafter = 500
+)
+
+// NewFactory creates a factory for Logging exporter
+func NewFactory() exporter.Factory {
+	return loggingexporter.NewFactoryWithName(typeStr)
```

Comment:
I think it's better to move all the code to the new exporter and make this call from the old one. WDYT? This PR can be kept as is to reduce the change set though.

Reply:
yeah I could go either way. I figured with this change, we could leave the code as is until its time to remove the logging exporter. at that point all the code could be moved. i just wanted to prevent having two copies of the same code.

One nice thing is with this change, once loggingexporter goes away, the NewFactoryWithName goes with it.

Reply:
> One nice thing is with this change, once loggingexporter goes away, the NewFactoryWithName goes with it.

I see. Ok, keeping it there is probably a better choice than 👍